bundle-workflow/src/run_bwc_test.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import logging
+import os
+import sys
+
+from manifests.bundle_manifest import BundleManifest
+from system.temporary_directory import TemporaryDirectory
+from test_workflow.bwc_test.bwc_test_suite import BwcTestSuite
+from test_workflow.test_args import TestArgs
+
+
+def main():
+    args = TestArgs()
+    with TemporaryDirectory(keep=args.keep) as work_dir:
+        logging.info("Switching to temporary work_dir: " + work_dir)
+        os.chdir(work_dir)
```

Comment:
Can use `with WorkingDirectory(work_dir) as cur_dir` here

Reply:
Sure will take care of this in a quick follow up PR.